mindsdb/integrations/handlers/aqicn_handler/tests/test_aqicn_handler.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+from mindsdb.integrations.handlers.aqicn_handler.aqicn_handler import AQICNHandler
+from mindsdb.integrations.handlers.aqicn_handler.aqicn_tables import *
+from mindsdb.api.executor.data_types.response_type import RESPONSE_TYPE
+
+
+class AQICNHandlerTest(unittest.TestCase):
```

Comment:
Better to add unit tests and mock the AQICNHandler calls

Reply:
You can check some of the tests here https://github.com/mindsdb/mindsdb/tree/main/tests/unit/handlers